src/Microsoft.AspNet.Authentication.OAuth/OAuthAuthenticationHandler.cs

```diff
@@ -159,14 +160,44 @@ protected override AuthenticationTicket AuthenticateCore()
             return new TokenResponse(oauth2Token);
         }
 
-        protected virtual async Task<AuthenticationTicket> GetUserInformationAsync(AuthenticationProperties properties, TokenResponse tokens)
+        protected virtual async Task<AuthenticationTicket> CreateTicketAsync(AuthenticationProperties properties, TokenResponse tokens)
```

Comment:
Well, the other notifications don't match the methods they are invoked from, so I'm not sure it's really important (`ApplyRedirect` from `ApplyResponseChallenge`, `ReturnEndpoint` from `InvokeReturnPathAsync`).

That said, if you really want me to, I'll change it ;)